src/cluster_slot_stats.c

```diff
@@ -143,7 +143,13 @@ void clusterSlotStatsAddNetworkBytesOutForReplication(int len) {
     if (c == NULL || !canAddNetworkBytesOut(c)) return;
 
     serverAssert(c->slot >= 0 && c->slot < CLUSTER_SLOTS);
-    server.cluster->slot_stats[c->slot].network_bytes_out += (len * listLength(server.replicas));
+    int64_t absolute_len = abs(len) * listLength(server.replicas);
+    if (len < 0) {
```

Comment:
Would this also be solved by just making len be a signed long long?

Reply:
Works but there is a catch. The underlying method applies `-` on a `sdslen` which returns `size_t` (long unsized int). I'm not sure if we should handle it explicit casting or implicit casting.

Submitting an update, let me know what you think.

Reply:
I'm okay with the proposal.